src/test/scala/examples/fancy_numbers/SparkTest.scala

```diff
@@ -1,68 +1,78 @@
 package examples.fancy_numbers
 
+import com.swoop.spark.SparkSessionTestWrapper
 import com.swoop.spark.records._
-import com.swoop.spark.test.SparkSqlSpec
 import org.apache.spark.sql.Dataset
 import org.apache.spark.storage.StorageLevel
 
 
-class SparkTest extends ExampleSpec with SparkSqlSpec with TestNegative5To100 {
+class SparkTest extends ExampleSpec with SparkSessionTestWrapper with TestNegative5To100 {
 
+  val sc = spark.sparkContext
   lazy val dc = SimpleDriverContext(sc)
   lazy val jc = dc.jobContext(SimpleJobContext)
   lazy val ds = recordsDataset(-5 to 100, jc)
   lazy val records = ds.collect
 
   "in an integration test" - {
     implicit val env = FlatRecordEnvironment()
-    val sqlContext = sqlc
-    import sqlContext.implicits._
+    import spark.implicits._
 
     behave like fancyRecordBuilder(records, jc)
 
     "should build records with Spark" in {
       ds.count should be(105)
     }
+
```

Comment:
No need for the extra whitespace for such simple tests: the description and `{ ... }` create sufficient visual separation without wasting vertical space.

Reply:
Yep, agreed, updated!